### DISCUSSION TITLE
ExpanderView, default open settings

### DISCUSSION THREAD

**Discussion by CyrilIlo34** [enisn/UraniumUI#736] [Ideas]
Hey , I'm currently using the last version of Uranium Ui and I needed to use the ExpanderView component.

But I'd like to know if it's possible to have a default open settings. In the doc, there is no such thing and it could be useful !

**Comment by enisn** [comment]
You can just set `IsExpanded` as `true` in XAML page. 

```xml
<uranium:ExpanderView IsExpanded="True">
...
```

**Reply by enisn** [reply]
I need to add this kind of properties in the documentation 👍
